server/seeds/index.js

```diff
@@ -1,11 +1,23 @@
 const db = require("../config/connection");
-const { Song } = require("../models");
+const { Song, User, Rating } = require("../models");
 const songSeeds = require("./songSeeds.json");
+const userSeeds = require("./userSeeds.json");
+const ratingSeeds = require("./ratingSeeds.json");
+const songs = Song.create(songSeeds);
 
 db.once("open", async () => {
   try {
     await db.dropDatabase();
-    await Song.create(songSeeds);
+    await songs;
+    const users = await User.create(userSeeds);
+    const newRating = await Rating.create({
+      rating: 3,
+      user: users[0]._id,
+      song: songs[0]._id,
+    });
+    await User.findByIdAndUpdate(users[0]._id, {
```

Comment:
This is a good start. If you add 2 or 3 more Ratings, it would make our app much easier to test.